app/src/main/res/values-sw360dp/auto_dimens.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <dimen name="sp_13">13sp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.sp_13` appears to be unused
 Unused resources make applications larger and slow down builds.